src/theory/strings/theory_strings_rewriter.h

```diff
@@ -540,6 +540,50 @@ class TheoryStringsRewriter {
                                      std::vector<Node>& approx,
                                      bool isOverApprox = false);
 
+  /**
+   * Checks whether it is always true that `a` is a strict subset of `b` in the
+   * multiset domain.
+   *
+   * Example:
+   *
+   * a = (str.++ "A" x), b = (str.++ "A" x "B") ---> true
```

Comment:
Perhaps use a more informative example, say `a = "A" ++ x, b = "B" ++ x ++ "AA"`

Reply:
I've added your example and a negative example as well.